### PR TITLE
2098 update time of day scale logic

### PR DESCRIPTION
**Pull request by tylerwhitson**
Fixes https://github.com/cityofaustin/atd-data-tech/issues/2098

- Updates scale so that 0 is always represented by the lowest color in the intensity range. The rest of the values are split up dynamically and represented by the remaining four colors in the color scheme.
- Adds a legend for better context

Screen Shot 2020-04-13 at 6 11 01 PM

Screen Shot 2020-04-13 at 6 11 14 PM

### CODE REVIEW

File: atd-vzv/src/App.css [hunk 1]
```diff
@@ -18,5 +18,6 @@ a:focus {
 }
 
 div.Tooltip-module_tooltip__omIxd {
+  color: #e4e5e6;
```
Comment (mateoclarke) [inline atd-vzv/src/App.css:21]: 🚨 rouge hex code 🚨 

this should probably live in a variable file for import.

File: atd-vzv/src/views/summary/CrashesByTimeOfDay.js [hunk 2]
```diff
@@ -100,6 +106,15 @@ const CrashesByTimeOfDay = () => {
             break;
         }
       });
+      // Set any 0 values to null so that the reaviz library
+      // recognizes them as "blank cells" and fills them accordingly
```
Comment (mateoclarke) [inline atd-vzv/src/views/summary/CrashesByTimeOfDay.js:110]: nice trick ✨

File: atd-vzv/src/views/summary/CrashesByTimeOfDay.js [hunk 3]
```diff
@@ -182,17 +202,43 @@ const CrashesByTimeOfDay = () => {
...
+                    }
+                  />
+                }
               />
             }
           />
         </Col>
       </Row>
+      <Row>
+        <Col className="pb-2">
```
Comment (mateoclarke) [inline atd-vzv/src/views/summary/CrashesByTimeOfDay.js:229]: Could we add some padding above the legend? maybe just update `pb-2` to `py-2`.

File: atd-vzv/src/views/summary/CrashesByTimeOfDay.js (hunk 2 above)
Comment (tylerwhitson) [reply atd-vzv/src/views/summary/CrashesByTimeOfDay.js:110]: Thanks!

File: atd-vzv/src/views/summary/CrashesByTimeOfDay.js (hunk 3 above)
Comment (tylerwhitson) [reply atd-vzv/src/views/summary/CrashesByTimeOfDay.js:229]: Done!

**Pull request merged**